src/SIM/Test/TestSIM.C

```diff
@@ -27,3 +51,93 @@ TEST(TestSIM, UniqueBoundaryNodes)
   std::sort(vec.begin(), vec.end());
   ASSERT_TRUE(std::unique(vec.begin(), vec.end()) == vec.end());
 }
+
+
+TEST_P(TestSIM2D, CreateDefaultGeometry)
+{
+  CreateDefaultGeometrySIM<SIM2D> sim(1);
+  TiXmlDocument doc;
```

Comment:
TestSIM2D er nødvendig for å bruke parametriserte tester med gtest. det er bare sånn det funker der, se instansering nederst.

jeg må ha en SIM fordi jeg skal teste protected members. så jeg eksponerer de via public members i wrapper-sim'en. det kommer mer tester etterhvert (senere commits), da gir det mer mening.